### DISCUSSION TITLE
Challenge Not Accepted

### DISCUSSION THREAD

**Discussion by BlockBoy32** [scaffold-eth/scaffold-eth-2#852] [Q&A]
Hi my ens is 0xblockboy.eth and I submitted challenge #1 6 days ago but it has not been accepted or rejected, can someone please help me? I completed #2 and am trying to keep moving forward but am currently blocked :(

**Comment by Pabl0cks** [comment]
Hello!

The day you submitted your challenge we had a problem with our autograder, and your submission got stuck, sorry 😅. 

Could you please resubmit it? Submissions should be assesed within minutes, if it ever stays in "Submitted" for more than 10 minutes, let us know!

Closing this, if you still have problems with your submission, feel free to reopen, or send us a message in the [Challenge 1 telegram](https://t.me/joinchat/E6r91UFt4oMJlt01)
